src/doc/rustc/src/platform-support/pc-windows-gnullvm.md

```diff
@@ -1,6 +1,6 @@
 # \*-pc-windows-gnullvm
 
-**Tier: 3**
+**Tier: 2 (without host tools)**
```

Comment:
Doesn't this need updating in https://doc.rust-lang.org/nightly/rustc/platform-support.html? Or are we auto-generating that?